tests/interfaces/default_qubit_2_integration/test_jax_jit_qnode_default_qubit_2.py

```diff
@@ -1825,14 +1825,23 @@ def circuit():
 
         assert jax.numpy.allclose(circuit(), jax.numpy.array([1.0, 0.0]))
 
-    @pytest.mark.xfail(
-        reason="Non-trainable parameters are not being correctly unwrapped by the interface"
-    )
+    # @pytest.mark.xfail(
+    #     reason="Non-trainable parameters are not being correctly unwrapped by the interface"
+    # )
```

Comment:
```suggestion
```

Reply:
wanted to keep this for context because i'm not entirely sure why certain cases still fail, and this comment might be helpful for someone in the future